backend/src/main/java/com/example/backend/model/Pet.java

```diff
@@ -0,0 +1,46 @@
+package com.example.backend.model;
+import org.springframework.data.annotation.Id;
+import org.springframework.data.mongodb.core.mapping.Document;
+
+import java.util.Arrays;
+import java.util.Objects;
+
+@Document("pets")
+public record Pet(
+        @Id
+        String id,
+        String name,
+        String nameOfBreed,
+        String photo,
+        String[] supplies) {
```

Comment:
You can use a List here, than you dont need to override methods ;-)

```suggestion
        List<String> supplies) {
```